contracts/src/v0.8/ccip/test/onRamp/OnRamp/OnRamp.forwardFromRouter.t.sol

```diff
@@ -212,32 +212,7 @@ contract OnRamp_forwardFromRouter is OnRampSetup {
     assertEq(IERC20(s_sourceFeeToken).balanceOf(address(s_onRamp)), feeAmount);
   }
 
-  function test_ShouldStoreNonLinkFees() public {
```

Comment:
This test produced stack too deep when running with via-ir and upon closer inspection it's a pretty worthless test. We transfer tokens manually and then check if that succeeded. The entire ccip call isn't even doing anything